databricks/assets/dashboards/databricks_cost_overview.json

```
@@ -1,747 +1,961 @@
-          "type": "toplist",
-          "requests": [{
-            "queries": [{
-              "data_source": "cloud_cost",
-              "name": "query1",
-              "query": "sum:all.cost{providername:Databricks,$job_id,$workspace_id,$servicename,$charge_description,$cluster_id} by {cluster_id}",
-              "aggregator": "sum"
-            }],
-            "response_format": "scalar",
-            "formulas": [{
-              "formula": "query1"
-            }],
-            "sort": {
-              "count": 500,
-              "order_by": [{
-                "type": "formula",
-                "index": 0,
-                "order": "desc"
-              }]
-            }
-          }],
-          "style": {
-            "display": {
-              "type": "stacked",
-              "legend": "automatic"
-            },
-            "palette": "datadog16"
+        "palette": "classic"
+       }
+      },
+      "layout": {
+       "x": 0,
+       "y": 4,
+       "width": 6,
+       "height": 3
+      }
+     },
+     {
+      "id": 3818638263356756,
+      "definition": {
+       "title": "Cost per Job Over Time",
+       "title_size": "16",
+       "title_align": "left",
+       "show_legend": false,
+       "legend_layout": "auto",
+       "legend_columns": ["avg", "min", "max", "value", "sum"],
+       "time": {
+        "hide_incomplete_cost_data": true
+       },
+       "type": "timeseries",
+       "requests": [
+        {
+         "formulas": [
+          {
+           "formula": "query1"
           }
-        },
-        "layout": {
-          "x": 0,
-          "y": 1,
-          "width": 6,
-          "height": 3
+         ],
+         "queries": [
+          {
+           "data_source": "cloud_cost",
+           "name": "query1",
+           "query": "sum:all.cost{providername:Databricks,$job_id,$workspace_id,$servicename,$charge_description,$cluster_id} by {job_id}.rollup(sum, daily)"
+          }
+         ],
+         "response_format": "timeseries",
+         "style": {
+          "palette": "dog_classic",
+          "order_by": "values",
+          "line_type": "solid",
+          "line_width": "normal"
+         },
+         "display_type": "line"
         }
-      }, {
-        "id": 427998264646722,
-        "definition": {
-          "title": "Cost per Cluster Over Time",
-          "title_size": "16",
-          "title_align": "left",
-          "show_legend": false,
-          "legend_layout": "auto",
-          "legend_columns": ["avg", "min", "max", "value", "sum"],
-          "time": {
-            "hide_incomplete_cost_data": true
+       ]
+      },
+      "layout": {
+       "x": 6,
+       "y": 4,
+       "width": 6,
+       "height": 3
+      }
+     }
+    ]
+   },
+   "layout": {
+    "x": 0,
+    "y": 17,
+    "width": 12,
+    "height": 8,
+    "is_column_break": true
+   }
+  },
+  {
+   "id": 3812785000101592,
+   "definition": {
+    "title": "Cluster Cost & Resource Utilization ",
```

Suggesting for consistency

```suggestion
    "title": "Cluster Cost and Resource Utilization ",
```